Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 /** Desired capabilities */
 def capabilities = [
   browserName: 'Firefox',
-  version: '51.0',
+  version: '52.0',
```

Comment:
Sauce Labs doesn't yet offer (or at least advertise) 53; they have dev and beta, though, the latter of which should map to 53, of course; not sure we want to use that, but it's worth a separate discussion!